pyleapProjects.json

```diff
@@ -166,6 +166,17 @@
           "clue_nrf52840_express"
       ]
     },
+	
+	{
+      "projectName": "PyLeap CLUE Barometer",
+      "projectImage": "https://cdn-learn.adafruit.com/assets/assets/000/115/821/medium640/projects_edited_P1390003.jpg?1665513267",
+      "description": "In this project, you'll turn your CLUE a barometric pressure and temperature monitor. A servo will display the pressure readings.",
```

Comment:
...turn your CLUE **into** a barometric...

Reply:
@BlitzCityDIY yes, when you can please rename the other project titles you've added